rasa/importers/importer.py

```diff
@@ -20,29 +19,26 @@ async def get_domain(self) -> Domain:
         """Retrieves the domain of the bot.
 
         Returns:
-            Loaded ``Domain``.
+            Loaded `Domain`.
         """
         raise NotImplementedError()
 
     async def get_stories(
         self,
-        interpreter: "NaturalLanguageInterpreter" = RegexInterpreter(),
```

Comment:
We are changing an interface here which might be re-implemented by a community member. Can we please add a removal changelog?